samples/perf_hgemm.cpp

```diff
@@ -223,7 +223,48 @@ using namespace rocwmma;
 *|_________|________________________________|
 */
 
-// Types
+namespace gfx9Params
+{
+    enum kernelParams : uint32_t
+    {
+        ROCWMMA_M = 32u,
+        ROCWMMA_N = 32u,
+        ROCWMMA_K = 16u,
+        BLOCKS_X  = 2u,
+        BLOCKS_Y  = 2u,
+        TBLOCK_X  = 128u,
+        TBLOCK_Y  = 2u,
+        WARP_SIZE = Constants::AMDGCN_WAVE_SIZE_64
+    };
+}
+
+namespace gfx11Params
+{
+    enum kernelParams : uint32_t
+    {
+        ROCWMMA_M = 16u,
+        ROCWMMA_N = 16u,
+        ROCWMMA_K = 16u,
+        BLOCKS_X  = 4u,
+        BLOCKS_Y  = 2u,
+        TBLOCK_X  = 64u,
+        TBLOCK_Y  = 4u,
+        WARP_SIZE = Constants::AMDGCN_WAVE_SIZE_32
+    };
+}
+
+#if (ROCWMMA_ARCH_GFX9)
+using namespace gfx9Params;
+#elif (ROCWMMA_ARCH_GFX11)
```

Comment:
Can probably reduce "elif" to "else" as both gfx11 and host will see the same params set. Even though the host doesn't use them per se, it is just there to satisfy the visibility by compiler.